_pages/faq.md

```diff
@@ -15,7 +15,7 @@ Making money is not a goal of the Catima project. Giving users a privacy-friendl
 
 ## Do you accept donations?
 
-If you feel like donating as a thank you, I accept donations through [GitHub Sponsors](https://github.com/sponsors/TheLastProject) and [PayPal](https://paypal.me/sylviavanos). But please do not feel pressured to :)
+Yes.  You can [make a donation](/donate) to show your appreciation for the project.
```

Comment:
Is there a reason for this extra space? Or is it a typo?

```suggestion
Yes. You can [make a donation](/donate) to show your appreciation for the project.
```

Reply:
Intentional in the sense that I'm one of those people using two spaces after a "." because I find that easier to read. Of course that only matters in monospace environments like my text editor (or e.g. man pages where this is fairly common); it will not make a difference to the rendered HTML.

Using it *here* was just force of habit as I usually try to stay consistent with the style used in the existing document. Nice catch, fixed.